packages/jetpack-blocks/package.json

```diff
@@ -37,20 +39,23 @@
     "swiper": "4.4.6"
   },
   "devDependencies": {
+    "@automattic/calypso-build": "file:../calypso-build",
+    "@automattic/calypso-color-schemes": "file:../calypso-color-schemes",
     "chalk": "^2.4.2",
     "copy-webpack-plugin": "^4.6.0",
     "enzyme": "^3.9.0",
     "enzyme-adapter-react-16": "^1.9.1",
     "enzyme-to-json": "^3.3.5",
     "react": "^16.8.3",
-    "webpack": "^4.29.6"
+    "webpack": "^4.29.6",
+    "webpack-cli": "3.2.3"
```

Comment:
Prolly needs a `^` here.

Reply:
Actually there's a mixture of strict and `^` here. I think we'll need `/.packages/.npmrc` to override default "strict" setting from the root config.

Reply:
https://github.com/Automattic/wp-calypso/pull/31503/commits/08b8be260d1b12ea564296c97040b2c1383d4d63

Reply:
>I think we'll need /.packages/.npmrc to override default "strict" setting from the root config.

Nope, not possible. :-( https://docs.npmjs.com/files/npmrc#files